lib/archethic/utxo/loader.ex

```diff
@@ -95,16 +96,14 @@ defmodule Archethic.UTXO.Loader do
          %ValidationStamp{
            protocol_version: protocol_version,
            ledger_operations: %LedgerOperations{
-             unspent_outputs: unspent_outputs,
-             consumed_inputs: consumed_inputs
+             unspent_outputs: unspent_outputs
            }
          },
-         phase2? \\ false
+         transaction_address
        ) do
     # Filter unspent outputs which have been consumed and updated (required in the AEIP21 Phase 1)
-    Enum.filter(unspent_outputs, fn %UnspentOutput{type: type} ->
-      phase2? or Enum.any?(consumed_inputs, &(&1.type == type))
-    end)
+    unspent_outputs
+    |> Enum.filter(&match?(%UnspentOutput{from: ^transaction_address}, &1))
```

Comment:
Since new token are coming from burn address, you should also filter the utxo with from == burn address

Reply:
New tokens are not coming from burn address, it's the reason to not cast them by default but use the burn only in the function to list consumed inputs

Reply:
New tokens are not coming from burn address, it's the reason to not cast them by default but use the burn only in the function to list consumed inputs